docs: add new Layout Functions: Center, Column and Row.. (#516)
ZupIT/beagle-docs: content/en/api/components/layout/margin.md

---
title: Margin
weight: 338
description: You will find here the description of the Margin function and its attributes details
---

---

## What is it?

> `Margin` is a function that applies an empty space that surrounds the component.

Reviewer: ```suggestion
`Margin` is a function to help you organize the layout of your view. It creates a space between the component passed as parameter and the outside content by setting its margins to the given value.
```

See how the structure is represented:

| **Attribute** | **Type**  | Required | **Definition** |
| :----------- | :------------------------------------------------------------- | :---------: | :---------------------------------------------------------------------------------------------------------------- |

> | self   | StyleComponent                                                |      ✓       | Component to be applied the style. |

Reviewer: ```suggestion
| self   | StyleComponent                                                |      ✓       | Component to receive the style. |
```

> | margin   | EdgeValue                                                |      ✓       | Value and position to be applied to the component. |

Reviewer: ```suggestion
| margin   | EdgeValue                                                |      ✓       | Margin to be applied to the component. |
```
